collectors/test-results/jira-xray/src/main/java/com/capitalone/dashboard/core/client/testexecution/TestExecutionClientImpl.java

```diff
@@ -322,17 +322,24 @@ private TestCase createTestCase(TestExecution.Test test, TestRun testRun, Featur
                 testCase.setStatus(TestCaseStatus.Unknown);
             }
 
-             Set<String> tags = getStoryIds(feature.getIssueLinks());
-            // Temporarily commented for core project update
-            // testCase.setTags(tags);
             testCase.setTestSteps(this.getTestSteps(testRun));
+            List<Feature> tc = featureRepository.getStoryByNumber(test.getKey());
+            if (tc.size() > 0) {
```

Comment:
Can you do a !CollectionUtils.IsEmpty instead of the > 0?

Reply:
updated as suggested